pod/annotations_test.go

```diff
@@ -20,7 +20,7 @@ func TestPodSchemaVersion(t *testing.T) {
 	}
 
 	ver, err := PodSchemaVersion(pod)
-	assert.NilError(t, err)
+	assert.Nil(t, err)
```

Comment:
`assert.NoError`

Reply:
the stretcher testify one doesn't have noError. Pretty sure nil is fine here.

Reply:
??

https://github.com/stretchr/testify/blob/master/assert/assertions.go#L1335

Reply:
Whoops!